src/elm/clinical.ts

```diff
@@ -139,10 +149,10 @@ export class CodeRef extends Expression {
   name: string;
   library: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.CodeRef) {
     super(json);
-    this.name = json.name;
-    this.library = json.libraryName;
+    this.name = json.name || '';
+    this.library = json.libraryName || '';
```

Comment:
Ack. Same deal as the logical specification stuff above (https://cql.hl7.org/04-logicalspecification.html#coderef)

Looks like I improperly marked these as required for the class